ci/testgrid/config.yaml

```diff
@@ -80,137 +77,104 @@ test_groups:
-- name: ci-knative-eventing-auto-release
-  gcs_prefix: knative-prow/logs/ci-knative-eventing-auto-release
 - name: ci-knative-eventing-dot-release
   gcs_prefix: knative-prow/logs/ci-knative-eventing-dot-release
+- name: ci-knative-eventing-auto-release
+  gcs_prefix: knative-prow/logs/ci-knative-eventing-auto-release
 - name: pull-knative-eventing-test-coverage
   gcs_prefix: knative-prow/logs/ci-knative-eventing-go-coverage
   short_text_metric: coverage
-
-# knative/eventing-sources
 - name: ci-knative-eventing-sources-continuous
   gcs_prefix: knative-prow/logs/ci-knative-eventing-sources-continuous
 - name: ci-knative-eventing-sources-nightly-release
   gcs_prefix: knative-prow/logs/ci-knative-eventing-sources-nightly-release
-- name: ci-knative-eventing-sources-auto-release
-  gcs_prefix: knative-prow/logs/ci-knative-eventing-sources-auto-release
 - name: ci-knative-eventing-sources-dot-release
   gcs_prefix: knative-prow/logs/ci-knative-eventing-sources-dot-release
+- name: ci-knative-eventing-sources-auto-release
+  gcs_prefix: knative-prow/logs/ci-knative-eventing-sources-auto-release
 - name: pull-knative-eventing-sources-test-coverage
   gcs_prefix: knative-prow/logs/ci-knative-eventing-sources-go-coverage
   short_text_metric: coverage
-
-# knative/docs
-- name: ci-knative-docs-continuous
-  gcs_prefix: knative-prow/logs/ci-knative-docs-continuous
-- name: pull-knative-docs-test-coverage
-  gcs_prefix: knative-prow/logs/ci-knative-docs-go-coverage
-  short_text_metric: coverage
-
-# knative/pkg
+- name: ci-knative-build-templates-continuous
+  gcs_prefix: knative-prow/logs/ci-knative-build-templates-continuous
 - name: ci-knative-pkg-continuous
   gcs_prefix: knative-prow/logs/ci-knative-pkg-continuous
 - name: pull-knative-pkg-test-coverage
   gcs_prefix: knative-prow/logs/ci-knative-pkg-go-coverage
   short_text_metric: coverage
-
-# knative/caching
 - name: ci-knative-caching-continuous
   gcs_prefix: knative-prow/logs/ci-knative-caching-continuous
 - name: pull-knative-caching-test-coverage
   gcs_prefix: knative-prow/logs/ci-knative-caching-go-coverage
   short_text_metric: coverage
-
-## release-0.2 branch
-
 - name: ci-knative-serving-0.2-continuous
   gcs_prefix: knative-prow/logs/ci-knative-serving-0.2-continuous
-
-## release-0.3 branch
-
 - name: ci-knative-serving-0.3-continuous
   gcs_prefix: knative-prow/logs/ci-knative-serving-0.3-continuous
-
-## release-0.4 branch
-
 - name: ci-knative-serving-0.4-continuous
   gcs_prefix: knative-prow/logs/ci-knative-serving-0.4-continuous
-
-#################################################################
-# Dashboards
-#################################################################
-
+######################################
 dashboards:
-
-## Master branch (head)
-
-# knative/serving
 - name: knative-serving
   dashboard_tab:
   - name: continuous
     test_group_name: ci-knative-serving-continuous
     base_options: 'sort-by-name='
+  - name: conformance-tests
```

Comment:
Can we keep the current order of tabs in Testgrid? If not 100%, at least most of it?

Reply:
Same as above.